_posts/2020-03-03-bazel-2-2.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: posts
+title: "Bazel 2.2"
+authors:
+  - philwo
+---
+
+[Bazel 2.2](https://github.com/bazelbuild/bazel/releases/tag/2.2.0) has just
+been released.
+
+Bazel 2.2 is intended to be backward-compatible with Bazel 2.0. There are no
+known breaking changes in this release.
+
+## Incompatible changes
+
+*   The flag `--incompatible_windows_bashless_run_command` has been removed. It
+    was flipped in Bazel 1.0
+*   The flag `--incompatible_windows_native_test_wrapper` has been removed. It
+    was flipped in Bazel 1.0
+*   The flag `--incompatible_disallow_dict_lookup_unhashable_keys` has been
+    removed.
+*   The flag `--incompatible_remap_main_repo` has been removed.
+*   The flag `--incompatible_disallow_unverified_http_downloads` has been
+    removed.
+*   Added the flag `--incompatible_applicable_licenses`. We plan to flip this
+    flag in Bazel 4.0. See
+    [#10687](https://github.com/bazelbuild/bazel/issues/10687) for details.
+*   Added the flag `--incompatible_remove_ram_utilization_factor`. We plan to
+    flip this flag in Bazel 3.0. See
+    [#10730](https://github.com/bazelbuild/bazel/issues/10730) for details.
+*   Added the flag
+    `--incompatible_load_proto_toolchain_for_javalite_from_com_google_protobuf`
+    to change the default value of `--proto_toolchain_for_javalite` to
+    `@com_google_protobuf//:javalite_toolchain`. We plan to flip this flag in
+    Bazel 3.0. See [#10335](https://github.com/bazelbuild/bazel/issues/10335)
+    for details.
+
+## cquery
+
+*   cquery's `config()` now supports arbitrary configurations.
+*   cquery: `//foo:bar` now means "all configured targets with label
+    `//foo:bar`" instead of "choose an arbitrary configured target with label
+    `//foo:bar`". See cquery docs for details.
+
+## Documentation
+
+*   `glob()` documentation is rewritten, and now it points out a pitfall of
+    rules shadowing glob-matched files.
```

Comment:
I don't think the change was really significant (just clarification of edge-cases in the documentation). We can remove it.